daphne/src/vdaf/prio3_test.rs

```diff
@@ -46,6 +46,30 @@ fn prepare_histogram() {
     .unwrap();
 }
 
+#[test]
+fn prepare_sum_vec() {
+    test_prepare(
+        &Prio3Config::SumVec { bits: 23, len: 3 },
+        DapMeasurement::U64Vec(vec![1, 0, 42]),
+        DapAggregateResult::U128Vec(vec![1, 0, 42]),
+    )
+    .unwrap();
+
+    test_prepare(
+        &Prio3Config::SumVec { bits: 23, len: 1 },
+        DapMeasurement::U64Vec(vec![1 << 23 - 1]),
```

Comment:
I believe this evaluates to `1<<22`: https://doc.rust-lang.org/reference/expressions.html#expression-precedence
```suggestion
        DapMeasurement::U64Vec(vec![(1 << 23) - 1]),
```